File: Pemrograman 1/MODUL 2/Bahasa Python/PRAK202-2410817110002-Akhmad_Daffa_Azzikri.py
nilai1 = float(input("Masukkan Nilai Pertama: "))
nilai2 = float(input("Masukkan Nilai Kedua: "))

hasil = nilai1 + nilai2

if int(nilai1) == nilai1:
    print(f'Hasil dari penjumlahan nilai pertama "{int(nilai1)}" dan nilai kedua "{nilai2:.1f}" adalah "{hasil:.2f}"')
else:
    print(f'Hasil dari penjumlahan nilai pertama "{nilai1:.2f}" dan nilai kedua "{nilai2:.1f}" adalah "{hasil:.2f}"')

File: Pemrograman 1/MODUL 2/Bahasa C/PRAK202-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>

int main() {
    float nilai1, nilai2, hasil;
    
    printf("Masukkan Nilai Pertama: ");
    scanf("%f", &nilai1);
    printf("Masukkan Nilai Kedua: ");
    scanf("%f", &nilai2);
    
    hasil = nilai1 + nilai2;

    if ((int)nilai1 == nilai1) {
        printf("Hasil dari penjumlahan nilai pertama \"%.0f\" dan nilai kedua \"%.1f\" adalah \"%.2f\"\n", nilai1, nilai2, hasil);
    } else {
        printf("Hasil dari penjumlahan nilai pertama \"%.2f\" dan nilai kedua \"%.1f\" adalah \"%.2f\"\n", nilai1, nilai2, hasil);
    }
    
    return 0;
}